### PR TITLE
Intro to GitHub apps

### PR DESCRIPTION
**Pull request by JasonEtco**
Adding some special interactions 🤖 

cc @hollenberry

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -60,4 +60,18 @@ module.exports = robot => {
...
+  const NAME = 'introduction-to-github-apps'
+  robot.on('installation_repositories.added', async context => {
+    const includes = context.payload.repositories_added.some(r => r.name === NAME)
+    if (!includes) return
+
+    return context.github.issues.create({
+      owner: context.payload.installation.account.login,
+      repo: NAME,
+      title: 'Great job installing WIP!',
+      body: 'Some body'
```
Comment (hiimbex) [inline index.js]: Are these arcuate?

```
title: 'Great job installing Request Info!',
body: 'Yay!!'
```

Comment (JasonEtco) [reply index.js]: Nope - @hollenberry needs to fix 'em

Comment (hollenberry) [reply index.js]: @JasonEtco can we still just add an issue comment on the open PR?

Comment (hollenberry) [reply index.js]: If so, can we set the body to... 

'Well done! You successfully installed the request info app

_disclaimer_ If you use this app in future repos, you won't get a message like this. This is just for Learning Lab!`

Or something similar.

cc: @a-a-ron for feedback

Comment (a-a-ron) [reply index.js]: Let me know if that works @JasonEtco so we can update this

**Comment by JasonEtco** [comment]
@hiimbex good to go 🚀

**Comment by hiimbex** [comment]
Thanks yall!

**Pull request merged**